plugins/tiddlywiki/ai-tools/globals.tid

```diff
@@ -12,12 +15,14 @@ resultTitlePrefix - Prefix of the tiddler to be used for saving the result. If t
 resultTags - Tags to be applied to the result tiddler
 ai-tools-status-title - Optional title of a tiddler to which the status of the request will be bound: "pending", "complete", "error"
 completionServer - Optional URL of server
+completionModel - Optional model to use
 -->
-\procedure ai-tools-get-llm-completion(conversationTitle,resultTitlePrefix,resultTags,ai-tools-status-title,completionServer)
+\procedure ai-tools-get-llm-completion(conversationTitle,resultTitlePrefix,resultTags,ai-tools-status-title,completionServer,completionModel,authTokenStoreKey)
```

Comment:
Keys should never been passed around as variables. You can pass around configuration tiddlers that tell the system, how to access keys. But keys need to be stored in the browser local storage and accessed on demand. Never copied to variables or tiddlers or fields.